astroid/manager.py

```diff
@@ -442,10 +442,12 @@ def clear_cache(self) -> None:
         # pylint: disable=import-outside-toplevel
         from astroid.brain.helpers import register_all_brains
         from astroid.inference_tip import clear_inference_tip_cache
+        from astroid.interpreter._import.spec import clear_spec_cache
         from astroid.interpreter.objectmodel import ObjectModel
         from astroid.nodes._base_nodes import LookupMixIn
         from astroid.nodes.scoped_nodes import ClassDef
 
+        clear_spec_cache()
```

Comment:
This can be moved to the list of lru_caches below, and then you can also remove the wrapper `clear_spec_cache()`.

Unless this is here just so it can be used in the unit tests to solve isolation issues? See next comment.